eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -367,14 +367,14 @@ decodeIntegerBounded64 = do
     TypeUInt64 -> pure ()
     TypeNInt -> pure ()
     TypeNInt64 -> pure ()
-    _ -> fail "expected major type 0 or 1 when decoding mint field"
+    _ -> fail "expected major type 0 or 1 when decoding smatbrMint field"
```

Comment:
I think it would be more readable when we left it to `mint` here

Reply:
Oops! Yes!